pkg/kubectl/cmd/get.go

```diff
@@ -312,9 +334,18 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 			}
 			lastMapping = mapping
 		}
-		if _, found := printer.(*kubectl.HumanReadablePrinter); found {
+		if resourcePrinter, found := printer.(*kubectl.HumanReadablePrinter); found {
+			resourceName := mapping.Resource
+			if alias, ok := kubectl.ResourceShortFormFor(mapping.Resource); ok {
+				resourceName = alias
+			} else if resourceName == "" {
+				resourceName = "n/a"
```

Comment:
Get rid of the `/`, maybe use "none" or "unknown". Otherwise kind/resource will look funny, e.g. `rc/n/a`